actfw_core/v4l2/video.py

```diff
@@ -870,10 +870,10 @@ def _ioctl(self, request, arg):
     def query_capability(self):
         cap = capability()
         result = self._ioctl(_VIDIOC.QUERYCAP, byref(cap))
-        if not (cap.capabilities & _V4L2_CAP_VIDEO_CAPTURE):
-            raise RuntimeError("The device doesn't support the single-planar API through the Video Capture interface.")
-        if not (cap.capabilities & _V4L2_CAP_STREAMING):
-            raise RuntimeError("The device doesn't support the streaming I/O method.")
+        if not (cap.device_caps & _V4L2_CAP_VIDEO_CAPTURE):
```

Comment:
In bullseye, multiple device files can be created for a single physical device. Only one of these device nodes can be used for video capture, so we have changed `capabilities` (physical device capabilities) to `device_caps` (device node capabilities).
This has been tested and confirmed to work on both buster and bullseye.

ref: https://www.kernel.org/doc/html/v5.15/userspace-api/media/v4l/vidioc-querycap.html#c.V4L.v4l2_capability